sklearn/manifold/spectral_embedding_.py

```diff
@@ -236,6 +236,9 @@ def spectral_embedding(adjacency, n_components=8, eigen_solver=None,
         warnings.warn("Graph is not fully connected, spectral embedding"
                       " may not work as expected.")
 
+    # When norm_laplacian=True csgraph_laplacian returns the symmetric
```

Comment:
I would remove this comment since this indicated in the documentation of `csgraph_laplacian`

Reply:
The documentation of csgraph_laplacian does not mention that it is the symmetric normalization. This is important for the algorithm, that's why I added it here. However, I did create a scipy issue to address this so it could be removed here.

Reply:
I removed the comment